app-admin/etcd-wrapper/files/etcd-wrapper

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
```

Comment:
we can say that about python too (when applicable) as we are the ones installing things! There is no reason to use env in this context.